llm-models/batch-inference/transformer-batch-inference.ipynb

```diff
@@ -0,0 +1,1458 @@
+     ]
+    }
+   ],
+   "source": [
+    "!nvidia-smi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "9033c85a-47bd-4cdf-a789-9f46ab5740a9",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "### Parameters\n",
+    "\n",
+    "The model name & tokenizer name below should be adjustable to most of the other models existing in the hugging face world.\n",
+    "\n",
+    "It would also make sense to change the prompt if there is a change to the model. This one is specifically designed for the LLAMA V2 model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "74fd51cb-d946-4d34-8635-f5470d1cfd35",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Model Params\n",
+    "MODEL_NAME = \"meta-llama/Llama-2-7b-chat-hf\"\n",
+    "MAX_NEW_TOKENS = 300\n",
+    "MIN_LENGTH = 0\n",
+    "REPETITION_PENALTY = 1.2\n",
+    "TEMPERATURE = 0.1\n",
+    "TOP_P = 0.9\n",
+    "TOP_K = 50\n",
+    "DO_SAMPLE = True\n",
+    "USE_CACHE = True\n",
+    "\n",
+    "# Tokenizer Params\n",
+    "TOKENIZER_NAME = MODEL_NAME\n",
+    "MAX_TOKENS = 2048\n",
+    "\n",
+    "# Run Params (How many articles to use)\n",
+    "MAX_EXAMPLES = 10000\n",
+    "\n",
+    "# Storage Params\n",
+    "STORAGE_PATH = \"/dbfs/llm-examples\"\n",
```

Comment:
that would be a bit tricky because we use to storage path both for storing the model files and data. Since this is a cluster, if we use the `local_disk0` of the driver, the workers will have a hard time accessing that. Thats why I defaulted to DBFS here